### DISCUSSION TITLE
付款成功后快半个月了收不到升级验证码

### DISCUSSION THREAD

**Discussion by Hounest** [getlantern/lantern-forum-cn#1019] [Q&A]
大概十月中旬在手机app上买的专业版，邮箱显示支付成功，但之后验证码一直发不到邮箱。在app内已经反映了几次问题并且重发了几遍，一直得不到解决。希望app运营团队尽快给予验证码，或者办理退款。邮箱：2303055317@qq.com

**Comment by MaxMKMax** [comment]
@Hounest 您好. 请您提供我支付单据. 我帮您提交问题上去. 谢谢.

**Comment by Hounest** [comment]
订单号： GPA.3315-5024-6856-93709
订购时间： 2024年10月19日 GMT+8 22:02:18
今天登录看到我的客户端已经升级，十分感谢。

Mike K ***@***.***> 于 2024年11月5日周二 下午11:44写道：
